Gems/EMotionFX/Code/Tests/UI/SkeletonOutlinerTestFixture.cpp

```diff
@@ -0,0 +1,172 @@
+        AZ::Data::AssetId actorAssetId("{5060227D-B6F4-422E-BF82-41AAC5F228A5}");
+        AZ::Data::Asset<Integration::ActorAsset> actorAsset =
+            TestActorAssets::CreateActorAssetAndRegister<SimpleJointChainActor>(actorAssetId, numJoints, "ClothColliderTestsActor");
+
+        CreateSkeletonAndModelIndices();
+
+        EXPECT_EQ(m_indexList.size(), numJoints);
+    }
+    void SkeletonOutlinerTestFixture::AddColliderViaAddComponentButton(QString label, QString subLevelLabel)
+    {
+        EXPECT_GT(m_indexList.size(), 3) << "Make sure to have a skeleton";
+        // Find the 3rd joint after the RootJoint in the TreeView and select it
+        SelectIndexes(m_indexList, m_treeView, 3, 3);
+
+        auto* treeView = GetAddCollidersTreeView();
+        auto* model = treeView->model();
+
+        //  find indices
+        QModelIndexList indices = model->match(
+            model->index(0, 0),
+            Qt::DisplayRole,
+            QVariant::fromValue(label),
+                    -1,
+            Qt::MatchRecursive);
+
+        if (subLevelLabel != "")
+        {
+            indices = model->match(
+                model->index(0, 0, indices[0]),
+                Qt::DisplayRole,
+                QVariant::fromValue(subLevelLabel),
+                    -1,
+                Qt::MatchRecursive);
+        }
+        //  check indices
+        if (indices.size() < 1)
+        {
+            EXPECT_GT(indices.size(), 0) << "Label not found";
+            return;
+        }
+        EXPECT_EQ(indices.size(), 1) << "Label is not eindeutig";
+
+        //  click first index
+        auto index = indices[0];
+        treeView->selectionModel()->select(index, QItemSelectionModel::Select | QItemSelectionModel::Rows);
+        treeView->clicked(index);
+    }
+
+    void SkeletonOutlinerTestFixture::ShowJointPropertyWidget()
+    {
+        auto* widget = GetJointPropertyWidget();
+        auto* mw = new QMainWindow;
+        mw->setFixedHeight(900);
+        mw->layout()->addWidget(widget);
+        widget->setFixedHeight(800);
+
+        mw->show();
+
+        // while (true)
+        QApplication::processEvents();
+    }
+
+    /*
+        Test Cases
+     */
+    TEST_F(SkeletonOutlinerTestFixture, AddClothCollider)
+    {
+        SetUpPhysics();
+
+        // Find the 3rd joint after the RootJoint in the TreeView and select it
+        SelectIndexes(m_indexList, m_treeView, 3, 3);
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Sphere");
+
+        ShowJointPropertyWidget();
+
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+    }
+
+    TEST_F(SkeletonOutlinerTestFixture, ChangeClothColliderValue)
+    {
+        SetUpPhysics();
+
+        AddColliderViaAddComponentButton("Add Cloth Collider", "Capsule");
+
+        // Check the node is in the ragdoll
+        EXPECT_TRUE(ColliderHelpers::NodeHasClothCollider(m_indexList[3]));
+
+        // get the widget
```

Comment:
In general we start with Upper case letter in comments
// Upper case
The comments added in this CR seems to have both upper case and lower case starts. Can you change them to start with Upper case?

Reply:
sure =)